src/front/spv/mod.rs

```diff
@@ -1014,7 +1070,7 @@ impl<I: Iterator<Item = u32>> Parser<I> {
 
                     let si_lexp = self.lookup_sampled_image.lookup(sampled_image_id)?;
                     let coord_lexp = self.lookup_expression.lookup(coordinate_id)?;
-                    let coord_type_handle = self.lookup_type.lookup(coord_lexp.type_id)?.handle;
+                    //let coord_type_handle = self.lookup_type.lookup(coord_lexp.type_id)?.handle;
```

Comment:
Nit: Can be removed?

Reply:
We'll likely needed when we implement the proper array layer support here. Spirv coordinates contain the array layer, and we need to extract it (from the last component), and use the rest for the coordinates, so we'll need to know their type.